File: Printpress.UI/Printpress_Angular/src/app/features/spare-parts/models/spare-part-selling-invoice-create.dto.ts
export interface SparePartSellingInvoiceLineCreateDto {
  sparePartItemId: string;
  quantity: number;
  unitPrice: number;
}

export interface SparePartSellingInvoiceCreateDto {
  clientName: string;
  invoiceDate: string;
  lines: SparePartSellingInvoiceLineCreateDto[];
}

File: Printpress.Application/SpareParts/DTO/SparePartSellingInvoiceCreateDto.cs
namespace Printpress.Application;

public class SparePartSellingInvoiceCreateDto
{
    public string ClientName { get; set; }
    public DateTime InvoiceDate { get; set; }
    public List<SparePartSellingInvoiceLineCreateDto> Lines { get; set; }
}

public class SparePartSellingInvoiceLineCreateDto
{
    public Guid SparePartItemId { get; set; }
    public decimal Quantity { get; set; }
    public decimal UnitPrice { get; set; }
}
